src/lib/features/events/event-store.ts

```diff
@@ -472,10 +478,14 @@ export class EventStore implements IEventStore {
             featureName: row.feature_name,
             project: row.project,
             environment: row.environment,
+            groupType: row.group_type,
+            groupId: row.group_id,
         };
     }
 
     eventToDbRow(e: IBaseEvent): Omit<IEventTable, 'id' | 'created_at'> {
+        const transactionContext = (this.db as any).userParams;
```

Comment:
maybe user defined type guard would be more elegant: https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards